RecoEgamma/EgammaIsolationAlgos/plugins/isolation_cones/PhotonPFIsolationWithMapBasedVeto.cc

```diff
@@ -132,9 +132,9 @@ bool PhotonPFIsolationWithMapBasedVeto::isInIsolationCone(const reco::CandidateP
   // dealing here with patObjects: miniAOD case
   if ( aspacked.get() )    
   {
-    inFootprint = isInFootprint(aspat_photonptr ->associatedPackedPFCandidates(),aspacked);
-    
-    //checking if the charged candidates come from the appropriate vertex
+    //inFootprint = isInFootprint(aspat_photonptr ->associatedPackedPFCandidates(),aspacked);
```

Comment:
identical result can be figured out from 
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_0_pre10/PhysicsTools/PatAlgos/plugins/PATPhotonSlimmer.cc#L132

It would be more practical to save the reco-to-packed associations in 
ReducedEGProducer so that they are not re-derived again in the slimmers and  other code similar to this one